pip_run/scripts.py

```diff
@@ -11,6 +11,11 @@
 from jaraco.context import suppress
 from jaraco.functools import compose
 
+try:
+    import tomllib  # type: ignore
+except ImportError:
```

Comment:
```suggestion
except ImportError:  # Python <3.11
```
Just so that it's easy to know why it's wrapped in a try-except.
Also, why are there `# type: ignore` comments? Would you consider specifying the error codes?

Reply:
Sure, no problem. If it needs something to satisfy the coverage tool, I'm OK with that as well.

The `# type: ignore` was something I found I needed to add because the type checker seemed to think I was redefining the name `tomllib`. I copied the approach here from another project (can't honestly remember which, probably pip). I'm honestly not that interested in spending any more time trying to placate a type checker over something which is a perfectly standard idiom, but if you (or anyone else) can tell me what you'd want me to change, I've no objection. Specifically, I don't know what error code I'd need to specify, so someone would have to tell me.

Reply:
I asked because when I ran latest mypy (strict) on version 3.8 I read that these `# type: ignore` comments are unused. I'll check how it looks in the 3.11+ versions.
The whole project doesn't satisfy strict mypy mode yet, but I want to make it so. That's reason why I am interested in that.